tests/sys/geom/class/eli/conf.sh

```diff
@@ -55,16 +71,29 @@ for_each_geli_config() {
 # func <cipher> <secsize>
 for_each_geli_config_nointegrity() {
 	func=$1
+	ciphers=aes-xts:128 aes-xts:256 \
```

Comment:
These don't get the `local` treatment but above does? I imagine the ones above don't technically need to be local, but maybe best they all are?